src/styles/variables.scss

```diff
@@ -23,3 +28,48 @@ $fs-20: 20px;
 $fs-22: 22px;
 $fs-24: 24px;
 $fs-36: 36px;
+
+/*Font weights*/
+$fw-light: 300;
+$fw-regular: 400;
+$fw-semibold: 500;
+$fw-bold: 700;
+
+/*Screen Sizes*/
+$screen-sm: 810px;
+$screen-md: 1130px;
+
+/*Shadow mixins*/
```

Comment:
Add these styles in `mixins.scss` file @live-wire

Reply:
👍